src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -16,6 +16,7 @@
 - **Line**: {line}
 - **In patch**: {in_patch}
 - **Third Party**: {third_party}
+- **Valid rule**: {valid_rule}
```

Comment:
Nit: "Valid" is extremely generic and thus not very informative here. ("In patch" or "Third-party" here are pretty obvious, but it's not clear what we mean by "valid" or "invalid" rule here without reading the code). I'd suggest a more precise / explicit name like `Forcibly disabled {force_disabled}`.